misk-redis/src/main/kotlin/misk/redis/Redis.kt

```diff
@@ -0,0 +1,19 @@
+package misk.redis
+
+/** A Redis client. */
+interface Redis {
+  /** Deletes a single key. */
```

Comment:
these methods should document what is being returned, especially when it's not totally obvious, like `del`